### DISCUSSION TITLE
How to add custom domain for cloud Budibase?

### DISCUSSION THREAD

**Discussion by sangdth** [Budibase/budibase#7825] [Help] [answered]
Hi, I've tried to search Google and here, but mostly can find only questions for self-hosted version. I can't find a way to add my custom domain for Budibase on cloud.

**Comment by shogunpurple** [comment, marked as answer]
Hi @sangdth - custom domains are not yet supported on cloud. This is something we are planning on introducing over the coming months, but I can't give you an exact timeline.

**Reply by sangdth** [reply]
Thank you. I'm really looking forward for this feature.
